app/client/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,31 @@
+import styles from './CourseCard.module.scss'
+import Image from "next/image";
+
+
+function CourseCard({ courseId, courseTitle, courseInfo, courseRating, courseImage, courseInstructor }) {
+    const ratingPercent = (courseRating*100)/5;
+    return (
+        <div className={styles.courseCard}>
+            <img
+                src={courseImage}
+                width="100%"
+                height="50%"
+                layout="responsive"
+            />
+            <div className={styles.courseCardInfo}>
+                <p className={styles.courseCardTitle}>{courseTitle}</p>
```

Comment:
Please make title h3 and change font-family to Merriweather to distinguish it from other text.